templates/etc/httpd/conf.d/_ssl_vhost.conf.erb

```diff
@@ -1,7 +1,9 @@
 # allow older yum clients to connect, see bz 647828
```

Comment:
Through some black magic that I didn't bother to figure out, it comes out indented correctly as is.